### ISSUE TITLE
m3u playlist support

### ISSUE THREAD

**Issue by r14c** [skyjake/lagrange#329]
i noticed that lagrange at least recognizes that `.m3u` files contain media, but apparently cannot play them. 

gemini://null.media/arconite/playlists/Todo_Tranqui.m3u

would there be any interest in getting this working? I could give a patch a shot if its wanted.

i have a small demo client here: gemini://mu.xj-ix.luxe/

**Comment by skyjake** [comment]
I agree this could be a worthwhile feature to have. (Perhaps even coupled with some audio player UI improvements.)

How do you see this working? Currently audio content is handled via inline players, where you can play, pause, and restart playback. Would an .m3u be treated similarly, so it opens the inline player UI and keeps playing through the listed URIs one by one? .m3u can also include metadata about the tracks, I suppose that should be shown somehow. Adding a "next track" button seems appropriate, but how about "previous track"?

**Comment by r14c** [comment]
yeah! adding a little track info display would be really nice. a basic version that just plays the list and allows next/previous would already be amazing though.
